### PR TITLE
Switching Font Awesome to SVG Icons

### PR DESCRIPTION
**Pull request by jpdevries**
see Thinkful-Ed/curric-fewd-001-v5#46
Thinkful-Ed/curric-fewd-001-v5#10

if merged also will need to be merged into `gh-pages` branch

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -40,6 +40,11 @@ a:hover {
   text-decoration: underline;
 }
 
+.icon {
+  width:1.4em;
```
Comment (aric87) [inline css/index.css:44]: style should be prop: val;

File: css/index.css [hunk 2]
```diff
@@ -67,14 +72,16 @@ a:hover {
...
+
 }
 
 .search-section button {
-  font-size: 17px;
-  height: 30px;
   background-color: white;
   border: 1px solid #DDDDDD;
   border-radius: 3px;
+
```
Comment (aric87) [inline css/index.css]: why the white space?

File: index.html [hunk 3]
```diff
@@ -30,7 +27,9 @@
   <section class="search-section">
     <input type="text" placeholder="Search software jobs" />
     <button class="search-button">
-      <span class="fa fa-search"></span>
+      <svg class="icon">
+        <use xlink:href="images/icons.svg#icon-search"></use>
```
Comment (aric87) [inline index.html:31]: why doesn't this one have a title?

Comment (jpdevries) [inline index.html:31]: Because it is decoration. Screen readers know it is a submit button regardless of inner HTML (submit attribute was addd in another commit or PR)

File: css/index.css (hunk 1 above)
Comment (jpdevries) [inline css/index.css:44]: huh?

Comment (aric87) [inline css/index.css:44]: there should be a space between the colon and the value, to conform to the rest of the curriculum

File: index.html (hunk 3 above)
Comment (aric87) [inline index.html:31]: Students won't know that. Mentioned in the other PR, and you can add it there, but you need to explain specifically when there should be and shouldn't be a title.

**Comment by jpdevries** [comment]
FontAwesome 5 will support SVG with an open source SVG framework. Eventually, we could add FontAwesome back with SVG usage.
